lib/json/test/valid/string_in_object.json

```diff
@@ -1 +1,3 @@
-{"key":"hello this is a string"}
\ No newline at end of file
+{
```

Comment:
It's the way JSON files needs to be formatted according to the convention.
+ it doesn't interfere with the content of the JSON file.